scripts/build.tokens.ckerc20.mjs

```diff
@@ -78,6 +79,37 @@ const ORCHESTRATOR_PRODUCTION_ID = Principal.fromText('vxkom-oyaaa-aaaar-qafda-c
 
 const DATA_FOLDER = join(process.cwd(), 'src', 'frontend', 'src', 'env');
 
+const LOGO_FOLDER = join(process.cwd(), 'src', 'frontend', 'src', 'icp-eth', 'assets');
+
+const saveTokenLogo = async (canisterId, name) => {
+	const logoName = name.toLowerCase().replace('ck', '').replace('sepolia', '');
+	const file = join(LOGO_FOLDER, `${logoName}.svg`);
+
+	if (existsSync(file)) {
+		return;
+	}
+
+	const agent = await createAgent({
+		identity: new AnonymousIdentity(),
+		host: 'https://icp-api.io'
+	});
+
+	const { metadata } = IcrcLedgerCanister.create({
+		agent,
+		canisterId
+	});
+
+	const data = await metadata({});
+
+	const logoData = data.find((item) => item[0] === 'icrc1:logo')[1].Text;
```

Comment:
I would handle the fact that `icrc1:logo` is `undefined`. It should not happens in production but, it's probably safer given that per definition the logo is not mandatory. 

Particularly for staging.

Maybe we just stacktrace a warning like `console.warning("No token found for ...")`?

Reply:
yes, always better to have the check